externalId-migration-util/.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
pls copy .gitignore from other repos. The .idea folder is not fit to be checked in.